releasenotes/notes/old-agent-8193550a76b21357.yaml

```diff
@@ -0,0 +1,7 @@
+---
+upgrade:
+  - |
+    tracer: support for Datadog Agent v5 has been dropped. Support was
+    technically dropped with v1.0 see
+    https://ddtrace.readthedocs.io/en/v1.0.0/versioning.html#release-support
+    for the supported Datadog Agent versions.
```

Comment:
`trace: Remove testing for Datadog Agent v5 support. Datadog Agent v5 is no longer supported since ddtrace==1.0.0`


?